### PR TITLE
Offscreen text varying per service

### PR DESCRIPTION
**Pull request by sareh**
Part of #663 

Adding Service context & consuming context in Brand, for offscreen text.

Test instructions:
Visit News article http://localhost:7080/news/articles/c0000000027o
The offscreen text for the Brand logo is `BBC News`
```
<span class="VisuallyHiddenText-sc-1elk1l5-0 gDNxin">BBC News</span>
```
Visit Persian article http://localhost:7080/news/articles/c0000000028o
The offscreen text for the Brand logo is `BBC News فارسی`
```
<span class="VisuallyHiddenText-sc-1elk1l5-0 gDNxin">BBC News فارسی</span>
```

- [x] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

**Comment by sareh** [comment]
@bbc/article-devs-news It would be good to get a review of the approach this PR is taking. 

CodeClimate wants me to abstract out the ServiceContext Provider with the value for News, since it is used in the unit tests. I could do that with a component:
```
const ServiceContextProvider = ({ children, service }) => (
  <ServiceContext.Provider value={services[service]}>
    {children}
  </ServiceContext.Provider>
);
```
However, this abstracts away the Context Provider's API, which I am loathe to do. Especially since many people are not familiar with it, due to it being a relatively new API in React. 
Let me know what you think.

**Review by pjlee11 — CHANGES_REQUESTED** [review]
Good work, looks like some of the duplication can be removed. Do agreed with not addressing the duplication  around the Context api

File: src/app/containers/Article/fixtureData.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  },
+  content: {
+    model: {
+      blocks: [
+        {
+          blockId: 'h-1',
+          type: 'headline',
+          model: {
+            blocks: [
+              {
```
[the remark this answers is not in the document]
Comment (pjlee11) [reply src/app/containers/Article/fixtureData.js]: Could the duplication here be mitigated by re-using the `/model/blocks` constants? EG: `blockContainingText` from https://github.com/bbc/simorgh/blame/82e40a366cda28c9e12a40db3883812f873518b9/src/app/models/blocks.js#L26

**Comment by sareh** [comment]
The outstanding codeclimate issues are not related to this PR's work. I suggest fixing them independently of this PR, since this has already got to be quite a large bit of work.

**Review by pjlee11 — DISMISSED** [review]
Happy for the outstanding code climate duplication warnings to be handled in a follow up PR as they are seperate to this issue - great work 👍

**Review by bcmn — DISMISSED** [review]

File: src/app/containers/Article/index.jsx [hunk 2]
```diff
@@ -48,15 +49,15 @@ class ArticleContainer extends Component {
...
+      return { amp, data, service };
     } catch (error) {
       console.log(error); // eslint-disable-line no-console
       return {};
     }
   }
 
   render() {
-    const { amp, data } = this.props;
+    const { amp, data, service } = this.props;
```
Comment (jtart) [inline src/app/containers/Article/index.jsx]: `service` needs to be included in `ArticleContainers`' propTypes.

File: src/app/lib/config/services/news.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export default {
```
Comment (jtart) [inline src/app/lib/config/services/news.js]: Can you extract these out to variables? If they are exported default in such a way, they will be named `default` in a stack-trace, which is harder to debug.

**Review by jtart — CHANGES_REQUESTED** [review]

File: src/app/containers/Article/index.jsx (hunk 2 above)
Comment (sareh) [reply src/app/containers/Article/index.jsx]: @jtart I've addressed the issues that you've raised.

**Review by jtart — DISMISSED** [review]

**Comment by sareh** [comment]
There was a merge conflict, so this PR will need reviews again. @pjlee11 @bcmn :)

**Review by bcmn — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**